core/src/main/java/org/mapfish/print/processor/ProcessorGraphNode.java

```diff
@@ -231,5 +186,47 @@ protected Values compute() {
                 return values;
               });
     }
+
+    private void executeProcess(
+        final Processor<In, Out> process, final Values values, final String timerName) {
+      final Timer.Context timerContext = this.node.metricRegistry.timer(timerName).time();
```

Comment:
use try ?

Reply:
The try will call close() which only calls stop() which is already called, so I don't see the point in calling close() after a stop()